src/Bicep.LangServer/Providers/AzureContainerRegistriesProvider.cs

```diff
@@ -51,8 +58,6 @@ await foreach (TenantResource tenant in tenants)
                 {
                     JArray jArray = JArray.Parse(data.ToString());
```

Comment:
This is definitely not for this PR, but I think we need to replace `Newtonsoft.Json` stuff with `System.Text.Json` types at some point.